src/go/k8s/pkg/resources/statefulset.go

```diff
@@ -103,6 +103,7 @@ type StatefulSetResource struct {
 	logger                   logr.Logger
 
 	LastObservedState *appsv1.StatefulSet
+	threshold         float64
```

Comment:
nit: this is very generic name. Treshold of WHAT? I think we should rename to be more specific

Reply:
Ok, renamed